libs/application/templates/children-residence-change/src/fields/ParentBIntro/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { useIntl } from 'react-intl'
+import { Box, Button, Text } from '@island.is/island-ui/core'
+
+import {
+  childrenResidenceInfo,
+  formatAddress,
+  getSelectedChildrenFromExternalData,
+} from '../../lib/utils'
+import { parentBIntro } from '../../lib/messages'
+import { CRCFieldBaseProps } from '../../types'
+import { DescriptionText } from '../components'
+
+const ParentBIntro = ({ field, application }: CRCFieldBaseProps) => {
+  const { externalData, answers } = application
+  const { formatMessage } = useIntl()
+  const applicant = externalData.nationalRegistry.data
+  const childResidenceInfo = childrenResidenceInfo(applicant, answers)
+  const children = getSelectedChildrenFromExternalData(
+    applicant.children,
+    answers.selectedChildren,
+  )
+  return (
+    <>
```

Comment:
Some stuff here could be extracted into components, not sure if we want to, what do you think @VilliGunn ?

Reply:
Yes we could extract some of this to a component to use on the overview screen and success screens. I suggest we do it when working on the success screen task.